<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CONTROL DE REGISTRO DE VENTAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtFecha">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>521</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>ÁREA DE VENTA</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccione producto:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccione tipo de pago:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboProducto">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Seleccione producto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Colección Escolar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Colección PreUniversitaria</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Colección Profesional</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cboPago">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Seleccione tipo de pago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Contado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tarjeta</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCantidad">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Precio del Producto</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPrecio">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>REGISTRAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNuevo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NUEVO</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblDatos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Producto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo Pago</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descuento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Recargo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio Final</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
